index.html

```diff
@@ -2,146 +2,196 @@
+          <li>bootstrap</li>
+          <li class="ruby">Ruby</li>
+        </ul>
+        <button class="button" type="button">See Project</button>
+      </li>
       <li>
         <div>
-
-          <img class="image1" src="./placeholder.png" alt="image placeholder">
+          <img class="image1" src="./placeholder.png" alt="image placeholder" />
         </div>
-          <h2 class="post">Multi-Post Stories</h2>
-              <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups
-                required. has been the industry's standard dummy
-                text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-              <ul class="lang">
-                <li>css</li>
-                <li>html</li>
-                <li>bootstrap</li>
-                <li class="ruby">Ruby</li>
-              </ul>
-              <button class="button" type="button">See Project</button>
-            </li>
+        <h2 class="post">Multi-Post Stories</h2>
+        <p class="text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="lang">
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li class="ruby">Ruby</li>
+        </ul>
+        <button class="button" type="button">See Project</button>
+      </li>
     </ul>
+  </section>
+  <section class="footer">
+    <div class=footer-text>
+      <h2 class="contact">CONTACT US</h2>
+      <p class="contact-text">&nbsp;&nbsp;&nbsp;&nbsp;If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+    </div>
+      <form action="https://formspree.io/f/xeqnejgd" method="post" class="conct-form">
+        <input type="text" name="name" id="username" placeholder="Text Input" maxlength="30" required /> <br>
+        <input type="email" name="email" id="useremail" placeholder="jeremy@gmail.com" required /><br>
+        <textarea name="message" id="usermessage" placeholder="Enter text here" maxlength="500" cols="30" rows="10"></textarea>
+        <button class="button" type="button">See Project</button>
+      </form>
+    
 
-      
   </section>
+  <hr class="line1" />
+  <ul class="icons">
+    <li>
+      <a href="#"><i class="fa fa-github bars"></i></a>
+    </li>
+    <li>
+      <a href="#"><i class="fa fa-twitter bars"></i></a>
+    </li>
+    <li>
+      <a href="#"><i class="fa fa-linkedin bars"></i></a>
+    </li>
+    <li>
+      <a href="#"><i class="fa fa-angellist bars"></i></a>
+    </li>
+    <li>
+      <a href="#"><i class="fa fa-medium bars"></i></a>
+    </li>
+  </ul>
+  <div class="mail-to">
+    Email nicomutale@gmail.com
+  </div>
```

Comment:
- [ ] Good job implementing the icons.   However, they are a little off to the right, and their order and separation are not the same as in the Figma.  Kindly correct them so they look more consistent with the given design.
- [ ] Kindly please remove the word "email" from your email, to make this part look like the Figma.
- [ ] The font-weight for the email should be "500".  You have "normal" (400).  Kindly please correct this one.

Your project:

![image](https://user-images.githubusercontent.com/57421823/149084805-bfea64c0-0af7-431a-bbc8-b4c9ba20975e.png)

The figma:

![image](https://user-images.githubusercontent.com/57421823/149084852-d3be8e8a-390c-408d-adf2-0068c4211d1b.png)

Reply:
- [x] This well addressed and resolved ✔️